collections/reports/labeljsongui.php

```diff
@@ -17,6 +17,7 @@
 	</head>
 	<body>
     <main>
+      <h1 class="page-heading">Label Content Format Visual Editor</h1>
```

Comment:
Turns out the the header was only part of the problem. I made some improvements in 29e06452e. What do you think now, @themerekat ?

Reply:
Looks pretty good now! Only two comments: change text to "Label Format Visual Editor"
and is it possible to expand this part so that you don't have scrollbars showing up for each line on the label?
![image](https://github.com/BioKIC/Symbiota/assets/16688784/c4cb55e3-2e7c-48c4-88b4-7279381298f6)